docs/reference/data-type-support.rst

```diff
@@ -0,0 +1,805 @@
+``HIPBLAS_USE_HIP_BFLOAT16`` before including the header file ``<hipblas.h>``.
+
+.. note::
+
+   The ``hip_bfloat16`` data type is only supported on AMD platforms.
+
+.. _hipblas_complex:
+
+Complex data types
+------------------
+
+hipBLAS uses the HIP types ``hipComplex`` and ``hipDoubleComplex`` in its API.
+
+Functions data type support
+=======================================
+
+This section collects the data type support tables of BLAS functions and
+solver functions. The cuBLAS backend does not support all the functions and all
+the types. For more information, see :ref:`cuBLAS backend <hipblas-backend>`.
+
+Level 1 functions - vector operations
+-------------------------------------
+
+Level-1 functions perform scalar, vector, and vector-vector operations.
+
+.. tab-set::
+
+  .. tab-item:: Float types
+    :sync: float-type
+
+    .. list-table::
+        :header-rows: 1
+        :widths: 15, 45, 10, 10, 10, 10
+
+        * 
+          - Function
+          - Description
+          - float16
+          - bfloat16
+          - float
+          - double
+
+        *
+          - :ref:`AMax <hipblas_amax>`, :ref:`AMin <hipblas_amin>`, :ref:`ASum <hipblas_asum>`
+          - Finds the first index of the element of minimum or maximum magnitude of a vector x or computes the sum of the magnitudes of elements of a real vector x.
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`AXPY <hipblas_axpy>`
+          - Scales a vector and adds it to another: :math:`y = \alpha x + y`
+          - ✅
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`Copy <hipblas_copy>`
+          - Copies vector x to y: :math:`y = x`
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`Dot <hipblas_dot>`
+          - Computes the dot product: :math:`result = x^T y`
+          - ✅
+          - ✅
+          - ✅
+          - ✅    
+
+        *
+          - :ref:`NRM2 <hipblas_nrm2>`
+          - Computes the Euclidean norm of a vector.
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`Rot <hipblas_rot>`, :ref:`Rotg <hipblas_rotg>`
+          - Applies and generates a Givens rotation matrix.
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`Rotm <hipblas_rotm>`, :ref:`Rotmg <hipblas_rotmg>`
+          - Applies and generates a modified Givens rotation matrix.
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`Scal <hipblas_scal>`
+          - Scales a vector by a scalar: :math:`x = \alpha x`
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        *
+          - :ref:`Swap <hipblas_swap>`
+          - Swaps corresponding elements of two vectors x and y: :math:`x_i \leftrightarrow y_i \quad \text{for} \quad i = 0, 1, 2, \ldots, n - 1`
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+  .. tab-item:: Complex types
```

Comment:
May be easier if these are just combined with the above table? Just a suggestion, I'm fine either way.

Reply:
We got a too long table, that's why I had to split. Selected the current form at rocBLAS:
https://rocm.docs.amd.com/projects/rocBLAS/en/develop/reference/data-type-support.html